Library/RewardCellProjectBackingStateType.swift

```diff
@@ -5,6 +5,7 @@ public enum RewardCellProjectBackingStateType: Equatable {
   public enum ProjectState {
     case nonLive
     case live
+    case successful
```

Comment:
It looks like we're using `successful` here to mean `post campaign pledging available`, which seems like a bad idea. That makes it easy to confuse with successful projects that don't have post campaign pledging enabled in the future. I think we should be consistent with the backend naming here and call this `inPostCampaignPledgingPhase` instead, which is a little long but much clearer.